src/main/webapp/js/user-page.js

```diff
@@ -222,9 +222,9 @@ function showProfileEditingForm(user) {
   const profileEditForms = document.getElementById('profile-edit-forms');
   const profileInfoForm = document.getElementById('profile-form');
   profileInfoForm.action += '/' + user.id;
-  profileEditForms.hidden = false;
   const profile = document.getElementById('profile');
   profile.hidden = true;
+  profileEditForms.hidden = false;
```

Comment:
Move this right after the init of `const profileEditForms`? That way you're grouping these statements by variable.